src/main/java/uk/gov/pay/products/config/ProductsModule.java

```diff
@@ -65,16 +65,18 @@ private JpaPersistModule jpaModule(ProductsConfiguration configuration) {
         final Properties properties = new Properties();
         properties.put("javax.persistence.jdbc.driver", dbConfig.getDriverClass());
         properties.put("javax.persistence.jdbc.url", dbConfig.getUrl());
-        properties.put("javax.persistence.jdbc.user", dbConfig.getUser());
-        properties.put("javax.persistence.jdbc.password", dbConfig.getPassword());
+        if (dbConfig.getUser() != null)
```

Comment:
We like curly braces around our `if` statements!

If you wanted to do it all on one line, you could do this:

```java
Optional.ofNullable(dbConfig.getUser).ifPresent(user -> properties.put("javax.persistence.jdbc.user", user));
```